commands/board/list.go

```diff
@@ -231,7 +231,7 @@ func Watch(instanceID int32, interrupt <-chan bool) (<-chan *rpc.BoardListWatchR
 	runErrs := dm.RunAll()
 	if len(runErrs) == len(dm.IDs()) {
 		// All discoveries failed to run, we can't do anything
-		return nil, fmt.Errorf("%v", runErrs)
+		return nil, &commands.UnavailableError{Message: tr("Error starting board discoveries"), Cause: runErrs[0]}
```

Comment:
I don't like that we only take the first `error`, we lose some information this way. 😕

Reply:
Fixed with a `fmt.Errorf("%v", errs)` for now, a better PR is coming after this one.